app/instrumentation-tests/src/org/commcare/utils/InstrumentationUtility.kt

```diff
@@ -310,6 +318,18 @@ object InstrumentationUtility {
         }
     }
 
+    fun verifyIntentTextOnSubmitReport(text : String){
```

Comment:
we should place this function inside `LogSubmissionTest` itself. We only add general purpose functions in this call which can be utilised across different tests and this function is very specific to the report problem test.

Reply:
I have added the steps inside the test itself and removed the function from here.